lib/polling.js

```diff
@@ -48,46 +52,62 @@ TradeOfferManager.prototype.doPoll = function(doFullUpdate) {
 	}
 
 	this.emit('debug', 'Doing trade offer poll since ' + offersSince + (fullUpdate ? ' (full update)' : ''));
-	this.getOffers(fullUpdate ? EOfferFilter.All : EOfferFilter.ActiveOnly, new Date(offersSince * 1000), (err, sent, received) => {
-		if (err) {
-			this.emit('debug', "Error getting trade offers for poll: " + err.message);
-			this.emit('pollFailure', err);
+
+	var filter = fullUpdate ? EOfferFilter.All : EOfferFilter.ActiveOnly;
```

Comment:
@Aareksio we could actually go further. ESLint enforces let/const usage instead of var. I disabled it, because I guess it's too strict. But anyway I think it's a good idea